### DISCUSSION TITLE
Is there any example for artery/aorta?

### DISCUSSION THREAD

**Discussion by zhang-qiang-github** [su2code/SU2#1866] [Q&A]
I want to implement a CFD for vessel, for example artery/aorta. 

I think it should be incompressible navier stokes simulation, am I right?

In `SU2/TestCases/incomp_navierstokes`, `bend/buoyancy_cavity/cylinder/streamwise_periodic` has only `cfg` file, but `.su2` file is lacked. 

Is there any example for a CFD simulation in artery or aorta? Any suggestion is appreciated~~~

**Comment by bigfooted** [comment]
Yes, for blood flow in the aorta the flow can be considered incompressible and usually laminar.
All the meshes are in a separate repository:
https://github.com/su2code/TestCases
If the walls of the aorta are considered rigid, then the simulation can be set up like a pipe flow.

**Reply by zhang-qiang-github** [reply]
Thanks for kindly reply. Is `TestCases/incomp_navierstokes/chtPinArray_3d` or `TestCases/incomp_navierstokes/pipeSlice_3d` the most similar demo for aorta? Or which demo is the most similar?

**Reply by zhang-qiang-github** [reply]
I find `incomp_navierstokes/streamwise_periodic` simulate a cylinder, and it is similar to aorta. I will use this demo in my application.

**Reply by zhang-qiang-github** [reply]
> If the walls of the aorta are considered rigid, then the simulation can be set up like a pipe flow.

@bigfooted I find the example of `streamwise_periodic` may be not suit for my aorta application, because the blood flow is not periodic. Is there any pipe flow example?

**Reply by bigfooted** [reply]
There is a laminar bend test case as well

**Reply by zhang-qiang-github** [reply]
Do you mean `TestCases/incomp_navierstokes/bend`? I have run it, and it can output pressure. Is it possible to output wall shear stress?
